### PR TITLE
[44355834] sync to ckan

### PR DESCRIPTION
**Pull request by m-rios**
# Description of feature
...

# Checklist
- [ ] (If applicable) added example values of new ENV variables to .env.local and explained them in README.md.
- [ ] Added unit tests to test the code.
- [ ] Added integration tests to test the code within the application as a whole.

### CODE REVIEW

File: app/use_cases/create_or_update_protocol.rb
Comment (emerencia) [inline app/use_cases/create_or_update_protocol.rb:83]:
```suggestion
    # empty push subscription definition, the existing push subscriptions would be deleted. This is intended, since this
```

**Comment by emerencia** [comment]
LGTM. I like that we're reusing stuff like push subscriptions for new projects.

**Review by emerencia — APPROVED** [review]

File: spec/controllers/questionnaire_controller_spec.rb
Comment (emerencia) [inline spec/controllers/questionnaire_controller_spec.rb:50]: I don't think rubocop will like this. 

Btw, you can have Rubymine's reformat fix this for you, if you have the alignment set up correctly (in settings, under editor, code style, ruby).

Comment (m-rios) [reply spec/controllers/questionnaire_controller_spec.rb:50]: my rubocop isn't complaining...

**Comment by m-rios** [comment]
Do not merge until base branch is merged https://github.com/compsy/u-can-act/pull/2512

**Pull request merged**
